example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -0,0 +1,220 @@
+package org.aerogear.mobile.example.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
+import android.widget.RadioButton;
+import android.widget.TextView;
+
+import org.aerogear.mobile.example.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+
+/**
+ * Fragment for show casing security service self-defence checks.
+ */
+
+public class SecurityServiceFragment extends BaseFragment {
+
+    View view;
+
+    @BindView(R.id.trustScore)
+    ProgressBar trustScore;
+
+    @BindView(R.id.trustScoreText)
+    TextView trustScoreText;
+
+    @BindView(R.id.trustScoreHeader)
+    TextView trustScoreHeader;
+
+    @BindView(R.id.rootAccess)
+    RadioButton rootAccess;
+
+    @BindView(R.id.lockScreenSetup)
+    RadioButton lockScreenSetup;
+
+    @BindView(R.id.emulatorAccess)
+    RadioButton emulatorAccess;
+
+    @BindView(R.id.debuggerAccess)
+    RadioButton debuggerAccess;
+
+    @BindView(R.id.hookingDetected)
+    RadioButton hookingDetected;
+
+    @BindView(R.id.allowBackup)
+    RadioButton allowBackup;
+
+    @BindView(R.id.deviceEncrypted)
+    RadioButton deviceEncrypted;
+
+    @BindView(R.id.deviceOS)
+    RadioButton deviceOS;
+
+    @BindView(R.id.developerOptions)
+    RadioButton developerOptions;
+
+
+    // Used to calculate trust store percentage
+    private float totalTests = 0;
+    private float totalTestFailures = 0;
+
+    @Override
+    public int getLayoutResId() {
+        return R.layout.fragment_security_service;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater,@Nullable ViewGroup container,
+                              @Nullable Bundle savedInstanceState) {
+        view = inflater.inflate(R.layout.fragment_security_service, container, false);
+        ButterKnife.bind(this, view);
+
+        runTests();
+
+        return view;
+    }
+
+    public void runTests() {
+
+        // perform detections
+        detectRoot();
+        detectDeviceLock();
+        detectEmulator();
+        debuggerDetected();
+        detectHookingFramework();
+        detectBackupEnabled();
+        detectDeviceEncryptionStatus();
+        detectLatestOS();
+        detectDeveloperOptions();
+
+        // get trust score
+        setTrustScore();
+    }
+
+    // tag::detectRoot[]
```

Comment:
I'm not sure if these asciidoc tags will be used with how document code for the 5.x stuff?